### PR TITLE
refactor(video-conferencing-service): modifiy naming convention of permission enums

### PR DESCRIPTION
**Pull request by bhavyasf**
SFO-95

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1299,184 +1299,6 @@
         "write-file-atomic": "^2.3.0"
       }
     },
-    "@loopback/boot": {
```
Comment (bhavyasf) [inline package-lock.json:1302]: happened when did `npm i`.

**Comment by samarpan-b** [comment]
Update description

File: services/video-conferencing-service/src/enums/permission-keys.enum.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 export const enum PermissionKeys {
-  CreateSession = 'CreateSession',
-  GenerateToken = 'GenerateToken',
-  StartArchive = 'StartArchive',
-  StopArchive = 'StopArchive',
-  GetArchives = 'GetArchives',
-  DeleteArchive = 'DeleteArchive',
+  CreateMeetingSession = 'CreateMeetingSession',
```
Comment (samarpan-b) [inline services/video-conferencing-service/src/enums/permission-keys.enum.ts:2]: u could have just changed the values here and kept everything else same. That would have meant less impact.

Comment (bhavyasf) [reply services/video-conferencing-service/src/enums/permission-keys.enum.ts:2]: Ok!

File: services/video-conferencing-service/src/enums/permission-keys.enum.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-  DeleteArchive = 'DeleteArchive',
+  CreateSession = 'CreateMeetingSession',
+  GenerateToken = 'GenerateMeetingToken',
+  StartArchive = 'StartMeetingArchive',
+  StopArchive = 'StopMeetingArchive',
+  GetArchives = 'GetMeetingArchives',
+  DeleteArchive = 'DeleteMeetingArchive',
   StopMeeting = 'StopMeeting',
-  SetUploadTarget = 'SetUploadTarget',
+  SetUploadTarget = 'SetMeetingUploadTarget',
```
Comment (bhavyasf) [inline services/video-conferencing-service/src/enums/permission-keys.enum.ts:9]: @samarpan-b done changes!

File: package-lock.json (hunk 1 above)
Comment (samarpan-b) [reply package-lock.json:1302]: revert this

**Pull request merged**